README.md

```diff
@@ -1,24 +1,151 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [📹 video presentation](#video-presentation)
+  - [:camera: screenshots](#screenshots)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-* Ruby version
+- [📝 License](#license)
 
-* System dependencies
+# 💰 Budget App <a name="about-project"></a>
 
-* Configuration
+> mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Database creation
+## 📹 video presentation <a name="Presentation"></a>
 
-* Database initialization
+[Video Presentation](https://www.loom.com/share/853d249f4fdc4c55b90b9c4b528c0d9e)
 
```

Comment:
- [ ] As suggested by the previous reviewer that your video presentation is good But I would like to recommend you you should also highlight some interesting pieces of code or something you built that you are very proud of as required for this capstone 👍
Kindly show your code from vscode and the feature from vs code which you are very proud of making 👍

Reply:
@HAMMAS-SALEEM please check i have recorded a new video

Reply:
@Chaseknis I hope you are doing well 👍

I have checked your new video and in that video, you have not shown any piece of code from your project which you are very proud of making. Kindly rerecord a video by showing a piece of code as well 👍 as suggested by the previous reviewer 😊